### DISCUSSION TITLE
scala-cli vs bloop

### DISCUSSION THREAD

**Discussion by tsxjsx** [scalameta/metals-vscode#1462] [Q&A]
plz explain its difference and thier purpose

**Comment by tgodzik** [comment]
Thanks for the question! Please take a look at the documentation of both and if you have specific questions do ask them here.

https://scala-cli.virtuslab.org/
https://scalacenter.github.io/bloop/

In short:
- Bloop is a build server that Metals often uses to build/run your code
- Scala CLI is a command line tool that can replace a build tool for smaller tasks, it also compiles using Bloop by default
